packages/waku/src/lib/types.ts

```diff
@@ -28,23 +40,20 @@ export type Unstable_HandleRequest = (
   ) & {
     req: Request;
   },
-  utils: {
-    renderRsc: RenderRsc;
-    renderHtml: RenderHtml;
-  },
+  utils: RenderUtils,
 ) => Promise<ReadableStream | Response | 'fallback' | null | undefined>;
 
-export type Unstable_HandleBuild = (utils: {
-  renderRsc: RenderRsc;
-  renderHtml: RenderHtml;
-  rscPath2pathname: (rscPath: string) => string;
-  generateFile: (
-    pathname: string,
-    req: Request,
-    renderBody: () => Promise<ReadableStream | string>,
-  ) => Promise<void>;
-  generateDefaultHtml: (pathname: string) => Promise<void>;
-}) => Promise<void>;
+export type Unstable_HandleBuild = (
+  utils: RenderUtils & {
```

Comment:
Let's not do the `RenderUtils` type refactor in this PR.
We'll make some changes with #1751, and could reconsider it afterwards.